app/assets/stylesheets/welcome.scss

```diff
@@ -0,0 +1,51 @@
+$gold: #F8A803;
+
+.welcome-wrapper {
+  background-image: image-url('landing-page-pic.jpg');
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+  height: 100%;
+}
+
+.welcome-top-bar {
+  background: rgba(0,0,0,0);
+}
+
+.top-bar-section ul li a{
```

Comment:
Selector should have depth of applicability no greater than 2, but was 4<br>Opening curly brace `{` should be preceded by one space